src/transformers/training_args.py

```diff
@@ -881,6 +884,18 @@ class TrainingArguments:
             )
         },
     )
+    use_torchdynamo: bool = field(
+        default=False,
```

Comment:
Surely there will be various variations of this coming in future, is it not?

So I think it'd be good from the get-going to plan this argument as `key:value` where the value can be a complex combo, With the first value being "autograd;nvfuser" perhaps?

And then I'd call it just `torchdynamo`, so:

```
--torchdynamo "autograd;nvfuser"
```

Unless this is the only option that will ever be used, but somehow I doubt that.

CCing: @Chillee

Reply:
Let's still allow for one simple option `--torchdynamo "autograd;nvfuser"` is not user friendly at all.